components/Cards.js

```diff
@@ -20,3 +20,78 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+const axiosPromise = axios.get('https://lambda-times-api.herokuapp.com/articles')
+        .then((res) => {
+
+            let javascript = res.data.articles.javascript
+            let bootstrap = res.data.articles.bootstrap
+            let technology = res.data.articles.technology
+            let jquery = res.data.articles.jquery
+            let node = res.data.articles.node
+
+            javascript.forEach((article) => {
+                const newCard = articleCard(article)
+                document.querySelector('.cards-container').appendChild(newCard)
+                console.log("this", article)
```

Comment:
once completed I would suggest taking out these excess `console.log`s out. They are helpful for debugging but not needed to pollute the user's console. If they have meaning that is one thing but these have no benefit and might actually be more confusing than useful for anyone but the developer.